app/models/concerns/fog_extensions/openstack/server.rb

```diff
@@ -1,13 +1,9 @@
 module FogExtensions
   module Openstack
     module Server
-      def self.prepended(base)
-        class << base
-          attr_reader :nics
-          attr_accessor :boot_from_volume, :size_gb, :scheduler_hint_filter
-          attr_writer :security_group, :network # floating IP
-        end
-      end
+      attr_reader :nics
```

Comment:
I think these changes are needed now just because we're OK with Ruby 2.0+, on 2.4 (at least) they are required.